modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java

```diff
@@ -367,6 +367,183 @@ public void testGetProps() throws Exception {
 		}
 	}
 
+	@Test
+	public void testGetPropsWithAssetCategoriesAndChildAssetCategoriesSortedByKey()
+		throws Exception {
+
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext(
+				_company.getCompanyId(), _company.getGroupId(),
+				_user.getUserId());
+
+		AssetVocabulary assetVocabulary =
```

Comment:
What do you think about creating a custom vocabulary, so you can delete it and the categories are deleted?

Reply:
yes, that'd be a little improvement, thanks